drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -519,14 +519,17 @@ function DragAndDropTemplates(configuration) {
         var is_item_placed = function(i) { return i.is_placed; };
         var items_placed = $.grep(ctx.items, is_item_placed);
         var items_in_bank = $.grep(ctx.items, is_item_placed, true);
-        var item_bank_properties = {};
+        var item_bank_properties = {
+            attributes: {
+                'role': 'group',
+                'aria-label': gettext('Item Bank')
+            }
+        };
         if (ctx.item_bank_focusable) {
-            item_bank_properties.attributes = {
-                'tabindex': 0,
-                'dropzone': 'move',
-                'aria-dropeffect': 'move',
-                'role': 'button'
-            };
+            item_bank_properties.attributes['tabindex'] = 0;
+            item_bank_properties.attributes['dropzone'] = 'move';
+            item_bank_properties.attributes['aria-dropeffect'] = 'move';
+            item_bank_properties.attributes['role'] = 'button';
```

Comment:
@e-kolpakov I just copied these properties from the zone template when making the item bank act like a droppable zone. We will be removing `role="button"` from zones and draggables for [TNL-6029](https://openedx.atlassian.net/browse/TNL-6029), so you can either delete this line now or leave it in to be removed in [TNL-6029](https://openedx.atlassian.net/browse/TNL-6029) - whatever you prefer.